### PR TITLE
[Ryujinx.Graphics.OpenGL] Address dotnet-format issues

### PR DESCRIPTION
**Pull request by TSRBerry**
Base PR: #4670

### CODE REVIEW

File: src/Ryujinx.Graphics.OpenGL/BackgroundContextWorker.cs [hunk 1]
```diff
@@ -9,12 +9,12 @@ unsafe class BackgroundContextWorker : IDisposable
...
-        private Thread _thread;
+        private readonly Thread _thread;
         private bool _running;
 
-        private AutoResetEvent _signal;
-        private Queue<Action> _work;
-        private ObjectPool<ManualResetEventSlim> _invokePool;
+        private readonly AutoResetEvent _signal;
+        private readonly Queue<Action> _work;
+        private readonly ObjectPool<ManualResetEventSlim> _invokePool;
```
Comment (AcK77) [inline src/Ryujinx.Graphics.OpenGL/BackgroundContextWorker.cs:17]: Should be a namespace issue

File: src/Ryujinx.Graphics.OpenGL/HwCapabilities.cs [hunk 2]
```diff
@@ -40,16 +40,18 @@ public enum GpuVendor
             Nvidia
         }
 
-        private static readonly Lazy<GpuVendor> _gpuVendor = new Lazy<GpuVendor>(GetGpuVendor);
+        private static readonly Lazy<GpuVendor> _gpuVendor = new(GetGpuVendor);
 
-        private static bool _isAMD   => _gpuVendor.Value == GpuVendor.AmdWindows || _gpuVendor.Value == GpuVendor.AmdUnix;
-        private static bool _isIntel => _gpuVendor.Value == GpuVendor.IntelWindows || _gpuVendor.Value == GpuVendor.IntelUnix;
+#pragma warning disable IDE0051, IDE0055 // Disable formatting
```
Comment (AcK77) [inline src/Ryujinx.Graphics.OpenGL/HwCapabilities.cs:45]: Useless

File: src/Ryujinx.Graphics.OpenGL/Pipeline.cs [hunk 3]
```diff
@@ -1701,10 +1705,9 @@ public void RestoreImages1And2()
 
         public bool TryHostConditionalRendering(ICounterEvent value, ulong compare, bool isEqual)
         {
-            if (value is CounterQueueEvent)
+            // Compare an event and a constant value.
+            if (value is CounterQueueEvent evt)
             {
-                // Compare an event and a constant value.
-                CounterQueueEvent evt = (CounterQueueEvent)value;
```
Comment (AcK77) [inline src/Ryujinx.Graphics.OpenGL/Pipeline.cs:1711]: Remove extra empty line

File: src/Ryujinx.Graphics.OpenGL/Queries/CounterQueue.cs [hunk 4]
```diff
@@ -148,14 +148,13 @@ public void QueueReset()
...
-                case CounterType.SamplesPassed: return QueryTarget.SamplesPassed;
-                case CounterType.PrimitivesGenerated: return QueryTarget.PrimitivesGenerated;
-                case CounterType.TransformFeedbackPrimitivesWritten: return QueryTarget.TransformFeedbackPrimitivesWritten;
-            }
-
-            return QueryTarget.SamplesPassed;
+                CounterType.SamplesPassed => QueryTarget.SamplesPassed,
+                CounterType.PrimitivesGenerated => QueryTarget.PrimitivesGenerated,
+                CounterType.TransformFeedbackPrimitivesWritten => QueryTarget.TransformFeedbackPrimitivesWritten,
+                _ => QueryTarget.SamplesPassed,
```
Comment (AcK77) [inline src/Ryujinx.Graphics.OpenGL/Queries/CounterQueue.cs:156]:
```suggestion
                _ => QueryTarget.SamplesPassed
```

**Review by AcK77 — CHANGES_REQUESTED** [review]

File: src/Ryujinx.Graphics.OpenGL/BackgroundContextWorker.cs (hunk 1 above)
Comment (TSRBerry) [reply src/Ryujinx.Graphics.OpenGL/BackgroundContextWorker.cs:17]: What do you mean?

Comment (AcK77) [reply src/Ryujinx.Graphics.OpenGL/BackgroundContextWorker.cs:17]: There was an error about `ObjectPool` which isn't here anymore

File: src/Ryujinx.Graphics.OpenGL/Sync.cs [hunk 5]
```diff
@@ -15,13 +15,13 @@ private class SyncHandle
         }
 
         private ulong _firstHandle = 0;
-        private ClientWaitSyncFlags _syncFlags => HwCapabilities.RequiresSyncFlush ? ClientWaitSyncFlags.None : ClientWaitSyncFlags.SyncFlushCommandsBit;
+        private static ClientWaitSyncFlags SyncFlags => HwCapabilities.RequiresSyncFlush ? ClientWaitSyncFlags.None : ClientWaitSyncFlags.SyncFlushCommandsBit;
```
Comment (AcK77) [inline src/Ryujinx.Graphics.OpenGL/Sync.cs:18]:
```suggestion
        private static ClientWaitSyncFlags _syncFlags => HwCapabilities.RequiresSyncFlush ? ClientWaitSyncFlags.None : ClientWaitSyncFlags.SyncFlushCommandsBit;
```

Comment (TSRBerry) [reply src/Ryujinx.Graphics.OpenGL/Sync.cs:18]: This is now a property so the default naming style applies again (https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/naming-rules#default-naming-styles). Should we still change it?

**Review by marysaka — APPROVED** [review]

File: src/Ryujinx.Graphics.OpenGL/Sync.cs (hunk 5 above)
Comment (gdkchan) [reply src/Ryujinx.Graphics.OpenGL/Sync.cs:18]: Is an answer for that question still pending? It seems it was already changed on other PRs.

**Review by gdkchan — APPROVED** [review]
lgtm, thanks.

File: src/Ryujinx.Graphics.OpenGL/Sync.cs (hunk 5 above)
Comment (TSRBerry) [reply src/Ryujinx.Graphics.OpenGL/Sync.cs:18]: Some final confirmation would be nice! :sweat_smile:

Comment (gdkchan) [reply src/Ryujinx.Graphics.OpenGL/Sync.cs:18]: I would prefer to follow the default naming guidelines. If it says it's PascalCase, then I think we should use PascalCase. The link you mentioned doesn't mention instance vs static members, so I assume it applies to both.

**Review by AcK77 — APPROVED** [review]

**Pull request merged**